ports/atmel-samd/boards/arduino_nano_33_iot/pins.c

```diff
@@ -0,0 +1,37 @@
+#include "shared-bindings/board/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
```

Comment:
The pin definitions here don't include unexposed pins that are connected to the WiFi module. I think it would make sense to add them. You could look at PyPortal and similar boards for the pin names we use, to make them consistent.

If you do work on this file again, could you reorder these to group the pins together that are aliases? For instance, A5 and SCL are the same. Look at some other board definitions for examples.

Reply:
"I think it would make sense to add them". I agree with @ladyada that it's going to be hard to do anything very interesting in the RAM and flash available, but something extremely minimal might be possible.

Reply:
> "I think it would make sense to add them". I agree with @ladyada that it's going to be hard to do anything very interesting in the RAM and flash available, but something extremely minimal might be possible.

Hello, it's weird not to have WiFi for a so-called IoT board, but using the onboard ATECC608A secure chip and LSM6DS3 IMU is useful enough to me and and probably for others.

Reply:
true, lets at least have the pins defined tho :)